Sources_v3/Controllers/ChannelController/ChannelController.swift

```diff
@@ -316,9 +316,9 @@ public class ChannelControllerGeneric<ExtraData: ExtraDataTypes>: Controller, De
                     $0.channelController(self, didReceiveMemberEvent: event)
                 }
             },
-            TypingEventObserver(notificationCenter: center, cid: channelId) { [unowned self] event in
+            TypingEventObserver(notificationCenter: center, cid: channelId) { [unowned self] _ in
                 self.delegateCallback {
-                    $0.channelController(self, didReceiveTypingEvent: event)
+                    $0.channelController(self, didChangeTypingMembers: self.channel?.typingMembers ?? [])
                 }
             }
```

Comment:
Should we still use the event observer for this? Shouldn't we rather use `onFieldChange`?

Reply:
I forgot we have such a wonderful thing! 😄 Fixed in b6946407c0255370ceaf38c856880e3c725a3770

Reply:
You built it!